.ci/scripts/utils.sh

```diff
@@ -40,6 +40,41 @@ install_pip_dependencies() {
   popd || return
 }
 
+install_domains() {
```

Comment:
These functions were removed by https://github.com/pytorch/executorch/pull/4826.  This was wrong for CI job and we need to bring them back.